Solutions/Corvus.Monitoring.AspnetCore.Mvc.Specs/Corvus.Monitoring.AspnetCore.Mvc.Specs.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(EndjinProjectPropsPath)" Condition="$(EndjinProjectPropsPath) != ''" />
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsPackable>false</IsPackable>
+    <!-- Disabling SA1204 because it prioritizes static/non-static over public/non-public, which doesn't fit very well
+         with bindings in SpecFlow.
+         Disabling SA1600, SA1602 (all public types and members to be documented) because test projects need to make lots of types
+         public for technical reasaons, but they are not meant for public consumption, so XML doc comments are only
```

Comment:
> reasaons

Typo!

Reply:
Lol. Copied and pasted this from the other specs project, so have corrected in those two locations too!